src/lib/numbers.js

```diff
@@ -0,0 +1,67 @@
+function add (a, b) {
+  const result = a + b;
+  return result;
```

Comment:
You can remove the declaration here and just return the result of the expression:
```
  return a + b;
```
You could also do this for all the other functions here